docs/source-2.0/aws/aws-iam.rst

```diff
@@ -434,6 +434,103 @@ The following example defines two operations:
     @actionName("OverridingActionName")
     operation OperationB {}
 
+.. smithy-trait:: aws.iam#conditionKeysResolvedByService
+.. _aws.iam#conditionKeysResolvedByService-trait:
+
+------------------------------------------------
+``aws.iam#conditionKeysResolvedByService`` trait
```

Comment:
What about `serviceResolvedConditionKeys` for the name of this trait? It's a clearer read to me and aligns more with the other IAM traits. Would need updating in several places (docs, code (ID, class names, etc.), test files.)

Reply:
renamed